hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/key/OMKeyRequest.java

```diff
@@ -307,6 +307,14 @@ protected List< OzoneAcl > getAclsForKey(KeyArgs keyArgs,
     return acls;
   }
 
+  protected static List<OzoneAcl> inheritDefaultAcls(KeyArgs keyArgs,
+      List<OzoneAcl> parentAcls) {
+    List<OzoneAcl> inheritAcls = new ArrayList<>();
+    inheritAcls.addAll(OzoneAclUtil.fromProtobuf(keyArgs.getAclsList()));
+    OzoneAclUtil.inheritDefaultAcls(inheritAcls, parentAcls);
+    return inheritAcls;
```

Comment:
Can be simplified, since `OzoneAclUtil.fromProtobuf` creates a new `ArrayList` for the ACLs from `keyArgs`:

```suggestion
    List<OzoneAcl> acls = OzoneAclUtil.fromProtobuf(keyArgs.getAclsList());
    OzoneAclUtil.inheritDefaultAcls(acls, parentAcls);
    return acls;
```